tests/test_adult.py

```diff
@@ -0,0 +1,52 @@
+import numpy
+
+from numpy.testing import assert_raises, assert_equal, assert_allclose
+
+from fuel.datasets import Adult
+from tests import skip_if_not_available
+
+
+def test_adult_test():
+    skip_if_not_available(datasets=['adult.hdf5'])
```

Comment:
In order for those tests not to be skipped, you'll need to add `adult` [here](https://github.com/mila-udem/fuel/blob/master/.travis.yml#L43), like so:

```
 - ./.travis-data.sh mnist binarized_mnist "caltech101_silhouettes 16" cifar10 cifar100 iris adult
```